docs/contributing/coding-standards/css.md

```diff
@@ -494,9 +494,20 @@ Good:
 
 More write up on [supported rules](https://stylelint.io/user-guide/rules/list).
 
-##  SassDoc
+## Comments
+
+You should normally use two slashes (`//`) at the start of the line for comments.
```

Comment:
```suggestion
For comments, you should normally use 2 slashes (`//`) at the start of the line.
```
I know this might look odd, so reason for suggestion: style guide says to write all numbers, except for one, as numerals.